tests/multiparser_js_css/__snapshots__/jsfmt.spec.js.snap

```diff
@@ -508,5 +526,33 @@ const Foo = styled.p\`
   }
 \`;
 
+styled.div\`
+  \${props => props.firstProp && css\`
+    color: red;
+  \`}
+\`;
+
+styled.div\`
+  \${props => props.firstProp &&
+    props.secondProp &&
+    props.thirdProp &&
+    css\`
+      color: red;
+    \`}
+\`;
```

Comment:
This doesn’t look as good to me. For reference, here’s what the current version of Prettier outputs:

**Prettier 1.19.1**
[Playground link](https://prettier.io/playground/#N4Igxg9gdgLgprEAuEBDARmAJnAZgOgHMALASwAMAdKSmAEmAAcAnCRgZwAIBeAPk5Zt2+XKWbsYABVaNOAMjkCZw9nEhQs0tvMWCO+GGWaaZOzmHbsqUTrduQANhGZJOzOFgDc1O+QC+PtbUIAA0IGwwpNDsyKCozKwA7pLxCDEoqABuEKRYoSDozKhgANZwMADKjMWkUITIMMwArnBhxDAAtg4A6mTw7NVgcBVppJGZYwCeyOCW+bWqzFJFhB2oyLioDqphAFbsAB4AQkWl5RWoHXAAMrVwG1s7IPsHFbWEDnAAik0Q8A-bVogariODMGboDBwBz5Fi1GDdXKGZAADgADGFBKpukVGDMWHBFpl7mEAI6-eBaDgzVDsAC0UDgHg8+Xc5LEcGkqFW6yQm0BYVUHVIDWaQPY70+Pz+9z5jyBMAwiKwyKQACYwo1UKQHO8AMIQDprGZQaAkkBNVQAFQw6X5T0yLQAkhoEJUwMxSIwYABBDQVGCTT4A1R+PxAA)
```sh
--parser babel
```

**Input:**
```jsx
abcdef.ghi`
	${props => props.firstProp && props.secondProp && props.thirdProp && css`
     color: red;
   `}
 `


```

**Output:**
```jsx
abcdef.ghi`
	${props =>
    props.firstProp &&
    props.secondProp &&
    props.thirdProp &&
    css`
      color: red;
    `}
 `;

```

Reply:
I've pushed a fix, it should be better now. Could you please have a look?

**Input:**
```javascript
styled.div`
  ${props => props.firstProp && css`
    color: red;
  `}
`

styled.div`
  ${props => props.firstProp && props.secondProp && props.thirdProp && css`
    color: red;
  `}
`
```

**Output**
```javascript
styled.div`
  ${props => props.firstProp && css`
    color: red;
  `}
`;

styled.div`
  ${props =>
    props.firstProp &&
    props.secondProp &&
    props.thirdProp &&
    css`
      color: red;
    `}
`;
```

Reply:
Much nicer!